Source/assembly/AssemblyInfoRF.cs

```diff
@@ -17,15 +18,19 @@
 // The form "{Major}.{Minor}.*" will automatically update the build and revision,
 // and "{Major}.{Minor}.{Build}.*" will update just the revision.
 
-[assembly: AssemblyVersion("@FILE_VERSION@")]
-[assembly: AssemblyInformationalVersionAttribute("@FULL_VERSION@")]
-[assembly: AssemblyFileVersion("@FILE_VERSION@")]
+[assembly: AssemblyVersion("1.0.0.0")]
+// [assembly: AssemblyInformationalVersionAttribute("@FULL_VERSION@")]
+#if CIBUILD
+[assembly: AssemblyFileVersion("@MAJOR@.@MINOR@.@PATCH@.@BUILD@")]
+#else
+[assembly: AssemblyFileVersion("14.6.0.0")]
+#endif
 
 // The following attributes are used to specify the signing key for the assembly,
 // if desired. See the Mono documentation for more information about signing.
 
 //[assembly: AssemblyDelaySign(false)]
 //[assembly: AssemblyKeyFile("")]
 
-[assembly: KSPAssembly("RealFuels", @MAJOR_VERSION@, @MINOR_VERSION@, @PATCH_VERSION@)]
+[assembly: KSPAssembly("RealFuels", 1, 0, 0)]
```

Comment:
This needs to be major/minor/patch as well--unless somehow github web diff is lying?

Reply:
Should be fixed with the latest commit